### ISSUE TITLE
Gem package error: Unable to build user-guide

### ISSUE THREAD

**Issue by jcanocan** [kubevirt/user-guide#582]
Hi,

I have been trying to `make build_img` the user-guide project. Unfortunately, the gem package outputs  the following error:

```
Retrying download gem from https://rubygems.org/ due to error (2/4): Encoding::UndefinedConversionError "\x8B" from ASCII-8BIT to UTF-8
Fetching premonition 2.0.1


Fetching unicode_utils 1.4.0

--- ERROR REPORT TEMPLATE -------------------------------------------------------


Gem::Package::FormatError: package metadata is missing in /usr/share/gems/cache/parallel-1.22.1.gem
  /usr/share/rubygems/rubygems/package.rb:670:in `verify_files'
  /usr/share/rubygems/rubygems/package.rb:596:in `block (2 levels) in verify'
  /usr/share/rubygems/rubygems/package/tar_reader.rb:27:in `new'
  /usr/share/rubygems/rubygems/package.rb:593:in `block in verify'
  /usr/share/rubygems/rubygems/package/file_source.rb:29:in `open'
  /usr/share/rubygems/rubygems/package/file_source.rb:29:in `with_read_io'
  /usr/share/rubygems/rubygems/package.rb:592:in `verify'
  /usr/share/rubygems/rubygems/package.rb:573:in `spec'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/rubygems_integration.rb:250:in `spec_from_gem'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/source/rubygems.rb:167:in `install'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/installer/gem_installer.rb:54:in `install'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/installer/gem_installer.rb:16:in `install_from_spec'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/installer/parallel_installer.rb:186:in `do_install'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/installer/parallel_installer.rb:177:in `block in worker_pool'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/worker.rb:62:in `apply_func'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/worker.rb:57:in `block in process_queue'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/worker.rb:54:in `loop'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/worker.rb:54:in `process_queue'
  /usr/share/gems/gems/bundler-2.2.33/lib/bundler/worker.rb:91:in `block (2 levels) in create_threads'


## Environment


Bundler       2.2.33
  Platforms   ruby, x86_64-linux
Ruby          3.0.4p208 (2022-04-12 revision 3fa771ddedac25560be57f4055f1767e6c810f58) [x86_64-linux]
  Full Path   /usr/bin/ruby
  Config Dir  /etc
RubyGems      3.2.33
  Gem Home    /usr/share/gems
  Gem Path    /root/.local/share/gem/ruby:/usr/share/gems:/usr/local/share/gems
  User Home   /root
  User Path   /root/.local/share/gem/ruby
  Bin Dir     /usr/bin
OpenSSL       
  Compiled    OpenSSL 1.1.1n  FIPS 15 Mar 2022
  Loaded      OpenSSL 1.1.1n  FIPS 15 Mar 2022
  Cert File   /etc/pki/tls/cert.pem
  Cert Dir    /etc/pki/tls/certs
Tools         
  Git         2.34.3
  RVM         not installed
  rbenv       not installed
  chruby      not installed


## Bundler Build Metadata


Built At          2022-09-09
Git SHA           unknown
Released Version  false


## Gemfile

### Gemfile

ruby
Encoding.default_external = Encoding::UTF_8
Encoding.default_internal = Encoding::UTF_8

source "https://rubygems.org"

gem "rake"
gem "html-proofer"
gem "premonition", "~> 2.0.1"
gem "unicode_utils"


### Gemfile.lock


<No /src/Gemfile.lock found>


--- TEMPLATE END ----------------------------------------------------------------

```

I have tried to build the project in kubuntu 20.04 and RHEL 8.6. Both systems throw the same error.
Please, let me know if more details are required.
Thanks!

**Comment by jcanocan** [comment]
I just found a [Dockerfile](https://github.com/cwilkers/project-infra/blob/b88b47b021a2ee31bd9eed1111b8d08555c3be0d/images/kubevirt-kubevirt.github.io/Dockerfile) fixing this issue. Very thanks to @cwilkers for fixing it. Also, I think it will be great to have this change in this repository as well. If you @cwilkers don't have time, I can do it for you. Of course I will reference you as the author of the change :)

**Comment by cwilkers** [comment]
I would like to create some issues around updating the container for this repo. 

There are two workflows (at a high level) affected by this change. One is the documentation author's experience with testing their changes before creating PRs, and the other is the back-end Prow jobs that test proposed PRs and eventually take merged PRs and translate those into updates to the website, 

Testing changes to the container used in Prow to actually generate the website does require a complicated local setup that I would like to better document.

Changing the local Makefile to use the kubevirt.github.io container (or Dockerfile) (found in project-infra) is a minor edit that should not interfere with the back end Prow jobs. (I did test against both repos as I updated the container, so it *should* be compatible.)
